templates/single-product/short-description.php

```diff
@@ -10,23 +10,25 @@
  * happen. When this occurs the version of the template file will be bumped and
  * the readme will list any important changes.
  *
- * @see 	    https://docs.woocommerce.com/document/template-structure/
- * @author 		WooThemes
- * @package 	WooCommerce/Templates
- * @version     1.6.4
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @author  Automattic
+ * @package WooCommerce/Templates
+ * @version 3.3.0
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly
+	exit; // Exit if accessed directly.
 }
 
 global $post;
 
-if ( ! $post->post_excerpt ) {
+$short_description = apply_filters( 'woocommerce_short_description', $post->post_excerpt );
+
+if ( ! $short_description ) {
 	return;
 }
 
 ?>
 <div class="woocommerce-product-details__short-description">
-    <?php echo apply_filters( 'woocommerce_short_description', $post->post_excerpt ); ?>
+	<?php echo $short_description; // WPCS: XSS ok. ?>
```

Comment:
@claudiosanches, wouldn't be better to use `esc_html()` instead of telling phpcs to ignore the XSS check?

Reply:
`woocommerce_short_description` applies some filters like `the_content`. That's why we are avoiding to escape things here.